baseplate/observers/tracing.py

```diff
@@ -257,9 +257,9 @@ def _create_binary_annotation(
         """
         endpoint_info = self._endpoint_info()
 
-        # Annotation values must be bool or str type.
-        if not isinstance(annotation_value, (bool, str)):
-            annotation_value = str(annotation_value)
+        # Annotation values must be str type.
+        if not isinstance(annotation_value, str):
+            annotation_value = str(annotation_value).lower()
```

Comment:
do we want things that weren't originally bools to get lowercased too?

Reply:
I'm not sure it matters too much but better safe than sorry.